migration_guide.md

```diff
@@ -2,7 +2,7 @@
 
 ## 0.29 to 0.30
 
-With this update, the minimum supported Rust version goes up to 1.69.
+With this update, the minimum supported Rust version goes up to 1.70.
```

Comment:
Please revert this change. It's for the `0.30` version. Since it's a MSRV update, it means next release will be `0.31`.

Reply:
done